app/test_split_integer.py

```diff
@@ -2,20 +2,22 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(6, 2)) == 6), \
+        "sum of parts must be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (split_integer(6, 2) == [3, 3]), "value divisible by parts"
```

Comment:
The parentheses around the assert condition are unnecessary. You can write `assert split_integer(6, 2) == [3, 3]` instead.